editor/store.js

```diff
@@ -20,13 +22,19 @@ const GUTENBERG_PREFERENCES_KEY = `GUTENBERG_PREFERENCES_${ window.userSettings.
 
 /**
  * Creates a new instance of a Redux store.
+ * @param {Object}        storePersistOptions Additional store persist options
  *
- * @return {Redux.Store} Redux store
+ * @return {Redux.Store}                      Redux store
  */
 function createReduxStore() {
 	const enhancers = [
 		applyMiddleware( multi, refx( effects ) ),
-		storePersist( 'preferences', GUTENBERG_PREFERENCES_KEY ),
+		storePersist( {
+			reducerKey: 'preferences',
+			storageKey: GUTENBERG_PREFERENCES_KEY,
+			defaults: STORE_DEFAULTS,
```

Comment:
If we're expecting the `storePersist` enhancer to handle persistence of a single reducer key, should we be passing the entire state object of defaults into it, or simply the default value of the persisted state subtree? This is made more complicated by the fact that we're assuming it'll be an object (nothing requiring it to be so, maybe it should at least be documented).